### ISSUE TITLE
Xaml (canvas) available without download

### ISSUE THREAD

**Issue by stefanolson** [Pictogrammers/pictogrammers.com#57]
### Describe your suggested feature

I always use the xaml (canvas) icons.  Was really good when I could just copy and paste it in the previous UI. Now I have to download it, which is much more time consuming.  Is it possible to put it back so can be viewed within the web page and copied without having to open it and then copy it?

**Comment by Orienser** [comment]
+1, please also add this feature to Android Vector Drawable. This allows me to easily use the icon without downloading it.

**Comment by mordfan** [comment]
YES! This was very convenient back then.

**Comment by njannink** [comment]
yes Im also missing the option to just copy the xaml path to clipboard
